src/config/normalize.js

```diff
@@ -1,12 +1,14 @@
-import { Dimensions } from 'react-native';
+import { Dimensions, PixelRatio } from 'react-native';
 
-// Default guideline sizes are based on standard ~6.12" screen mobile device (iPhone 14 Pro)
```

Comment:
Please do not remove this line but add the base device, where the guideline base width and height are from.

Reply:
Thanks, adjusted in the newest commit: https://github.com/smart-village-solutions/smart-village-app-app/pull/1127/commits/33d36bc8117f6958b9f362b64f91ec57a05d4cb1